### ISSUE TITLE
The result using this code base is much lower than that showed in the paper

### ISSUE THREAD

**Issue by jerry-lcj** [CAMMA-public/rendezvous-in-time#6]
Hi, I am trying to reproduce the result of this code base, I use cholect50 as data and follow the data split instructions. The train.sh is used as follow:
#! /bin/bash

CUDA_VISIBLE_DEVICES=0, python3 main.py \
    --exp_name rit_train \
    --log_name rit_train.log \
    --max_epochs 40 \
    --early_stopping_patience 5 \
    --fold 1 \
    --split 'cholect45-crossval' \
    --m 6 \
    --bs 16 \
    --nw 6 \
    --topK 5 \
    --ln 0 \
    --cg 0 \
    --od1 1e-6 \
    --od2 1e-6 \
    --od3 1e-6 \
    --mom 0.95 \
    --ms1 20 \
    --ms2 39 \
    --ms3 60 \
    --g1 0.94 \
    --g2 0.95 \
    --g3 0.99 \
    --layers 8 \
    --data_dir "path-to-CholecT50" \

Plus I also change the learning rate to 1e-3 to avoid the issue of under fitting. But i only got the IVT mAP as 0.1. Is there any training details i missed?
